runtime/commonMain/src/kotlinx/serialization/internal/AbstractPolymorphicSerializer.kt

```diff
@@ -81,7 +81,8 @@ public abstract class AbstractPolymorphicSerializer<T : Any> internal constructo
     public open fun findPolymorphicSerializer(
         decoder: CompositeDecoder,
         klassName: String
-    ): KSerializer<out T> = decoder.context.getPolymorphic(baseClass, klassName)
+    ): DeserializationStrategy<out T> = decoder.context.getPolymorphic(baseClass, klassName)
```

Comment:
It is not supported for sealed classes explicitly, WDYT?

Reply:
It should be supported: sealed class may have abstract inheritor, for whom we want to use 'default' serializer strategy